index.bs

```diff
@@ -4648,71 +4659,154 @@ The <dfn export for=commands>browsingContext.setViewport</dfn> command modifies
    </dd>
 </dl>
 
-<div algorithm="remote end steps for browsingContext.setViewport">
+<div algorithm>
```

Comment:
I think this helper algorithms could be moved outside of the command section so that remote end steps follow the CDDL type definitions for the command.

Reply:
Yeah, I guess, but we also have the same with "browsingContext.locateNodes" or "script.callFunction"